src/Slider.jsx

```diff
@@ -474,26 +500,38 @@ class Rheostat extends React.Component {
     killEvent(ev);
   }
 
-  // istanbul ignore next
-  handleSlide(x, y) {
-    const { orientation, onSliderDragMove } = this.props;
-    const { slidingIndex: idx, sliderBox } = this.state;
+  positionPercent(x, y, sliderBox) {
+    const { orientation } = this.state;
+    if (orientation === VERTICAL) {
+      return ((y - sliderBox.top) / sliderBox.height) * PERCENT_FULL;
+    }
+    const percent = ((x - sliderBox.left) / sliderBox.width) * PERCENT_FULL;
+    return percent;
```

Comment:
tiny comment: can we just return the previous line? This interim variable seems unnecessary